documentation/src/main/asciidoc/topics/config_examples/server_truststore_realm.xml

```diff
@@ -5,18 +5,18 @@
       <security-realm name="default">
          <server-identities>
             <ssl>
-               <!-- Provides an SSL server identity with a keystore that contains server certificates. -->
+               <!-- Adds a Java keystore that contains server certificates and encrypts client connections. -->
                <keystore path="server.p12"
                          relative-to="infinispan.server.config.path"
                          keystore-password="secret"
                          alias="server"/>
-               <!-- Provides a trust store that verifies client identities to the server. -->
+               <!-- Configures a trust store that contains client certificates or part of a certificate chain. -->
                <truststore path="trust.p12"
                            relative-to="infinispan.server.config.path"
                            password="secret"/>
             </ssl>
          </server-identities>
-         <!-- Authenticates client identities against the trust store, if required. -->
+         <!-- Validates client identities if the client trust store contains all public client certificates. -->
```

Comment:
"Validates client identitites against the truststore. If configured, the truststore must contain the public certificates of all clients."